s4e-backend/src/main/java/pl/cyfronet/s4e/controller/InvitationController.java

```diff
@@ -0,0 +1,83 @@
+package pl.cyfronet.s4e.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.web.bind.annotation.*;
+import pl.cyfronet.s4e.bean.Institution;
+import pl.cyfronet.s4e.controller.request.CreateInvitationRequest;
+import pl.cyfronet.s4e.controller.response.BasicInstitutionResponse;
+import pl.cyfronet.s4e.event.OnConfirmInvitationEvent;
+import pl.cyfronet.s4e.event.OnRejectInvitationEvent;
+import pl.cyfronet.s4e.event.OnSendInvitationEvent;
+import pl.cyfronet.s4e.service.InvitationService;
+
+import javax.validation.Valid;
+import javax.ws.rs.NotFoundException;
```

Comment:
Have a look at other places we use NFE, we have an exception definied in our project with the same name, i.e. pl.cyfronet.s4e.ex.NotFoundException.